feature/pom.xml

```diff
@@ -110,7 +110,6 @@ limitations under the License.
                 <feature>jclouds-compute</feature>
                 <feature>jclouds-api-atmos</feature>
                 <feature>jclouds-api-byon</feature>
-                <feature>jclouds-api-cloudsigma2</feature>
```

Comment:
I will try to add `super-features`, currently I'm struggling with docker and bouncycastle deps. cannot write a good feature for `jclouds-docker` yet